core/kits/documents/tabs/theme-style-images.php

```diff
@@ -200,16 +200,14 @@ protected function register_tab_controls() {
 		$this->add_control(
 			'image_hover_transition',
 			[
-				'label' => esc_html__( 'Transition Duration', 'elementor' ) . ' (s)',
+				'label' => esc_html__( 'Transition Duration', 'elementor' ),
 				'type' => Controls_Manager::SLIDER,
-				'range' => [
-					'px' => [
-						'max' => 3,
-						'step' => 0.1,
-					],
+				'size_units' => [ 's', 'ms' ],
+				'default' => [
+					'unit' => 's',
```

Comment:
@rami-elementor BC?

Reply:
The old selector had hard-coded unit `{{SIZE}}s`. For BC, I replaced the selector with `{{SIZE}}{{UNIT}}` and added a default of `'unit' => 's'`.